.github/workflows/test.yml

```diff
@@ -39,19 +44,25 @@ jobs:
           name: replay-driver
           path: macOS-recordreplay.so
   preview-branch:
-    name: Wait for Vercel Preview Branch
+    name: Vercel Preview Build
     runs-on: ubuntu-latest
     steps:
-      - name: Waiting for 200 from the Vercel Preview
-        uses: patrickedqvist/wait-for-vercel-preview@v1.1.1
-        id: waitFor200
+      - uses: actions/checkout@v2
+      - uses: amondnet/vercel-action@v20
+        id: vercel-deploy
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          max_timeout: 500
-          check_interval: 30
-      - run: echo ${{steps.waitFor200.outputs.url}}
+          # for posting a comment upon completion
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          vercel-token: "aLNsahI9JClTxsQuxV7Kf5SD"
```

Comment:
Let's make sure we revoke this token if it's useful for anything.

Reply:
It's my personal token, if we end up using this, we'd probably want to generate some more stable token and shove all of those keys into GH secrets.